index.bs

```diff
@@ -9951,9 +9951,12 @@ Dictionary {{AudioWorkletNodeOptions}} Members</h6>
 		{{AudioWorkletNode}}. If the string key of an entry in the
 		list does not match the name of any {{AudioParam}} objects in
 		the node, it is ignored.
+
 	: <dfn>processorOptions</dfn>
 	::
-		 This holds any additional user-defined data that may be used to initialize the corresponding {{AudioWorkletProcessor}} for this {{AudioWorkletNode}}.
+		This holds any additional user-defined data that may be used to
+		initialize the corresponding {{AudioWorkletProcessor}} for this
+		{{AudioWorkletNode}}.
```

Comment:
What changed here?

Reply:
There was a redundant space and it was a single long line. I think Bikeshed rendered this line incorrectly because of the space. Also I have been doing line-wrapping as much as possible.

Reply:
Please don't do that. It makes reviewing harder.  And also keep in mind:  https://github.com/tabatkins/bikeshed/issues/1470.  Until that's fixed, we should be very careful about wrapping lines.